core/esmf-test-aspect-models/src/main/resources/valid/samm_1_0_0/org.eclipse.esmf.test/1.0.0/AspectWithCharacteristicWithSeeAttribute.ttl

```diff
@@ -35,7 +35,6 @@
    samm:dataType xsd:string .
 
 :TestCharacteristicTwo a samm:Characteristic ;
-   samm:name "TestCharacteristicTwo" ;
```

Comment:
I think this should not be removed, since in SAMM 1.0.0 based Aspect Models, `samm:name` is required for every element.